src/web/js/output-ui.js

```diff
@@ -178,11 +178,11 @@
       Position.fromPyretSrcloc = function (runtime, srcloc, loc, documents, options) {
         return runtime.ffi.cases(isSrcloc, "Srcloc", loc, {
           "builtin": function(_) {
-             throw new Error("Cannot get Position from builtin location", loc);
+             return new Error("Cannot get Position from builtin location", loc);
```

Comment:
These seem like weird changes to have in the images update.

Reply:
I mostly agree.  However, IIRC the commit that introduced that (https://github.com/brownplt/code.pyret.org/commit/1eced4f62b1c15e35b20bf93b3d8893b4231a80f) was trying to recover from some weird stack issue that triggered in deeply-nested images when rendered in error messages, or something like that, and it boiled down to fixing the stack-unsafety of this case.  This change should be cherry-picked anyway, regardless of this PR.